adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
 def draw_letters():
-    pass
+    letters = {
+            'A': 9, 
+            'B': 2, 
+            'C': 2, 
+            'D': 4, 
+            'E': 12,           
+            'F': 2, 
+            'G': 3, 
+            'H': 2, 
+            'I': 9, 
+            'J': 1, 
+            'K': 1, 
+            'L': 4, 
+            'M': 2, 
+            'N': 6,     
+            'O': 8, 
+            'P': 2, 
+            'Q': 1, 
+            'R': 6, 
+            'S': 4, 
+            'T': 6, 
+            'U': 4, 
+            'V': 2, 
+            'W': 2, 
+            'X': 1, 
+            'Y': 2, 
+            'Z': 1
+    }
+
+    drawn_letter = []
+    all_list = letters.items()
+    mynew_list = list(all_list)
+    
+    while len(drawn_letter) < 10:
+        random_number = random.randrange(0,len(mynew_list))
```

Comment:
Small comment here, so in the project description it says that certain letters should have a higher probability of being chosen than other letters. For example, `E` should have the highest probability to be chosen since the `LETTER_POOL` says that there are 12 of them (`line 8`). What you have on `lines 33-34` don't necessarily take those weights into account. All letters have an equal chance of being chosen. A solution that considers those weights could look like this:
```python
def draw_letters():
   drawn_letters = []
   letters = []
   for letter in LETTER_POOL.keys():
       letters += letter * LETTER_POOL[letter]

   while True:
    if len(drawn_letters) == 10: break

    index = random.randint(0,len(letters) - 1)
    chosen_letter = letters[index]

    if drawn_letters.count(chosen_letter) < LETTER_POOL[chosen_letter]:
        drawn_letters.append(chosen_letter)

   return drawn_letters
```
In the beginning of the function we loop through the keys in `LETTER_POOL` to create a list with their appropriate weights.  `letters += 'S' * 4` --> `letters = ['S', 'S', 'S', 'S']`